sky/cli.py

```diff
@@ -3985,6 +4052,449 @@ def spot_dashboard(port: Optional[int]):
+        record = global_user_state.get_service_from_name(service_name)
+        if record is None:
+            with ux_utils.print_exception_no_traceback():
+                raise ValueError(f'\nService {service_name!r} not found.')
+        service_endpoint = record['endpoint']
+        if service_endpoint is None:
+            with ux_utils.print_exception_no_traceback():
+                raise ValueError(
+                    f'Endpoint not found for service {service_name!r}. '
+                    'Please check whether the service is ready.')
+        click.echo(service_endpoint)
+        return
+    query_services: Optional[List[str]] = None
+    if service_names:
+        query_services = _get_glob_services(service_names)
+    service_records = core.serve_status(query_services)
+    click.echo(f'{colorama.Fore.CYAN}{colorama.Style.BRIGHT}Services'
+               f'{colorama.Style.RESET_ALL}')
+    status_utils.show_service_table(service_records, all)
+    click.echo(f'\n{colorama.Fore.CYAN}{colorama.Style.BRIGHT}'
+               f'Replicas{colorama.Style.RESET_ALL}')
+    replica_infos = []
+    for service_record in service_records:
+        for replica_record in service_record['replica_info']:
+            # Only print FAILED replicas if:
+            # 1. --all is specified;
+            # 2. auto_restart is not enabled (in which FAILED replica count
+            #    as one replica).
+            if (all or not service_record['auto_restart'] or
+                    replica_record['status'] != serve_lib.ReplicaStatus.FAILED):
+                replica_record['service_name'] = service_record['name']
+                replica_infos.append(replica_record)
+    status_utils.show_replica_table(replica_infos, all)
+
+
+@serve.command('down', cls=_DocumentedCodeCommand)
+@click.argument('service_names',
+                required=False,
+                type=str,
+                nargs=-1,
+                **_get_shell_complete_args(_complete_service_name))
+@click.option('--all',
+              '-a',
+              default=None,
+              is_flag=True,
+              help='Stop all existing clusters.')
+@click.option('--yes',
+              '-y',
+              is_flag=True,
+              default=False,
+              required=False,
+              help='Skip confirmation prompt.')
+@click.option('--purge',
+              '-p',
+              is_flag=True,
+              default=False,
+              required=False,
+              help='Ignore errors (if any). ')
+def serve_down(
+    service_names: List[str],
+    all: Optional[bool],  # pylint: disable=redefined-builtin
+    yes: bool,
+    purge: bool,
+):
+    """Teardown service(s).
+
+    SERVICE_NAMES is the name of the service (or glob pattern) to tear down. If
+    both SERVICE_NAMES and ``--all`` are supplied, the latter takes precedence.
+
+    Tear down a service will delete all associated resources, including the
+    controller VM and all replicas.
```

Comment:
```suggestion
    Tearing down a service will delete all of its replicas and associated resources.
```
(controller VM will not be torn down now)

Reply:
Good catch! It is a remnant. Thanks!